test/helper.js

```diff
@@ -71,9 +71,9 @@ exports.withServer = (mock, skipping) => {
     exports.load.cfg('taskcluster.rootUrl', helper.rootUrl);
 
     helper.connect = bindings => {
-      let json = urlencode(JSON.stringify(bindings));
+      let jsonBindings = urlencode(JSON.stringify(bindings));
       debug('Connecting to api...');
-      const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', '/connect/?bindings=')+json);
+      const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', '/connect/?bindings=')+jsonBindings);
```

Comment:
Are you able to use template strings here?

```js
const evtSource = new EventSource(libUrls.api(helper.rootUrl, 'events', 'v1', `/connect/?bindings=${jsonBindings}`));
```

Reply:
Yes. Done!